index.html

```diff
@@ -27,27 +27,29 @@
 
     <div class="supercontainer col-12">
         <img  src="css/Craig-Titus-720x520.jpg" alt="Titus" class="bigpic col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
-        <div class="text-block col-6 sm-col-8 md-col-4 lg-co3 xl-col-3">
-          <h2>Quit smoking</h2>
+        <div class="text-block col-6 sm-col-6 md-col-6 lg-col-44 xl-col-4">
+          <h1>Quit smoking</h2>
           <p>Before it's too late</p>
         </div>
       </div>
-      <form class="contactform col-6 sm-col-4 md-col-4 lg-col-3 xl-col-3" >
-        e-mail<br>
-        <input type="text" name="email" class="email">
-        <input type="submit" value="QUIT" class="quit">
-      </form>
-      <aside class="col-6 sm-col-4 md-col-4 lg-col-3 xl-col-3 ">
-        <h3>Quitting smoking is hard, but quit-smoking aids can boost your chance of success.</h3>
+      
+      <section class="main-wrapper col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
+      <aside class="col-6 sm-col-4 md-col-4 lg-col-6 xl-col-6">
+        <h3 class="blackh  black">Quitting smoking is hard, but quit-smoking aids can boost your chance of success.</h3>
         <p>
             Want to stop smoking? Several quit-smoking products approved by the Food and Drug Administration (FDA) can help you stop for good.
             
             Using smoking-cessation products can greatly increase your chance of success. Only about 5 percent of people who try to quit tobacco succeed without a quit-smoking product. But more than 30 percent succeed when using one.
             
             Your chance of successfully quitting is even better when you combine behavior therapy with one or more quit-smoking products. You're more than twice as likely to quit smoking if you use prescription medication and professional support than if you try to quit on your own.</p>
-      </aside>
-     <main class="6-col">
-        Tools & Tips  How To Quit  Using Nicotine Replacement Therapy
+            <form class="contactform col-12 " >
+              e-mail<br>
+              <input type="text" name="email" class="email">
+              <input type="submit" value="QUIT" class="quit">
+            </form>
+          </aside>
+     <main class="col-7 sm-col-7 md-col-7 lg-col-9 xl-col-9">
+        <h3 class="tooltips">Tools & Tips  How To Quit  Using Nicotine Replacement Therapy</h3>
         Using Nicotine Replacement Therapy
```

Comment:
You should wrap this content in a div and use a tags to help you with the format of the content . An example of what you could use is here - https://www.w3schools.com/html/html_formatting.asp